### PR TITLE
TreeView Visual Update

### PR DESCRIPTION
**Pull request by karenbtlai**
## Description

Update TreeView to spec

## Screenshots (if appropriate):

image
image

### CODE REVIEW

File: dev/TreeView/TreeViewItem.xaml [hunk 1]
```diff
@@ -16,14 +16,18 @@
...
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="local:TreeViewItem">
-                    <Grid x:Name="ContentPresenterGrid" Margin="0,0,0,0" 
+                    <Grid x:Name="ContentPresenterGrid"
                         Background="{TemplateBinding Background}" 
                         BorderBrush="{TemplateBinding BorderBrush}"
                         BorderThickness="{TemplateBinding BorderThickness}"
-                        contract7Present:CornerRadius="{TemplateBinding CornerRadius}">
+                        Height="{TemplateBinding MinHeight}"
```
Comment (ranjeshj) [inline dev/TreeView/TreeViewItem.xaml:27]: >Height [](http://example.com/codeflow?start=24&length=6)

Will this force the item to be minheight and not let us size to content ?

File: dev/TreeView/TreeViewItem.xaml [hunk 2]
```diff
@@ -109,6 +116,20 @@
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
 
+                        <Grid
+                            HorizontalAlignment="Left"
+                            VerticalAlignment="Center">
```
Comment (ranjeshj) [inline dev/TreeView/TreeViewItem.xaml:121]: Is this Grid needed ? can you set the alignment properties on the Rectangle instead ?

File: dev/TreeView/TreeViewItem.xaml (hunk 1 above)
Comment (karenbtlai) [reply dev/TreeView/TreeViewItem.xaml:27]: The `MinHeight` setter property is currently set to `TreeViewItemMinHeight` so one should be able to size the content by overwriting that resource.

File: dev/TreeView/TreeViewItem.xaml (hunk 2 above)
Comment (karenbtlai) [reply dev/TreeView/TreeViewItem.xaml:121]: you're right. Removed.

File: dev/TreeView/TreeViewItem.xaml (hunk 1 above)
Comment (karenbtlai) [reply dev/TreeView/TreeViewItem.xaml]: Oh, realized in the `TreeViewItemTemplateSelectorTest` that it is unreasonable to expect developers to set `MinHeight` to resize content. I've tweaked the template using padding on `ContentPresenterGrid` to get the correct item height.

File: dev/TreeView/TreeViewItem.xaml [hunk 3]
```diff
@@ -120,6 +141,7 @@
                                 <CheckBox x:Name="MultiSelectCheckBox"
                                             Width="32"
                                             MinWidth="32"
+                                            MinHeight="{ThemeResource TreeViewItemMinHeight}"
```
Comment (ranjeshj) [inline dev/TreeView/TreeViewItem.xaml:144]: You should name this TreeViewItemMultiSelectCheckBoxMinHeight now that this is only for the checkbox.

**Review by ranjeshj — APPROVED** [review]
:shipit:

**Pull request merged**